tools/visualModelEditor/README.md

```diff
@@ -0,0 +1,63 @@
+# Visual Model Editor
+
+A visual diagramming tool for graphically desigining an RSDL/OData entity data model.
+
+![Screenshot](./screenshot.jpeg)
+
+## Installation and build instructions
+
+Run
+
+```
+npm install
+```
+
+Then run:
+
+```
+npm run build
+```
+
+The build command will store the output files in the `dist` folder.
```

Comment:
Note: `npm run build` produces warnings - not sure if they can simply be ignored. Should be mentioned here, or better make it build without warnings.

~~~sh
WARNING in ./node_modules/handlebars/lib/index.js 22:38-56
require.extensions is not supported by webpack. Use a loader instead.
 @ ./src/templates.ts 7:37-58
 @ ./src/index.ts 5:20-42

WARNING in ./node_modules/handlebars/lib/index.js 23:2-20
require.extensions is not supported by webpack. Use a loader instead.
 @ ./src/templates.ts 7:37-58
 @ ./src/index.ts 5:20-42

WARNING in ./node_modules/handlebars/lib/index.js 24:2-20
require.extensions is not supported by webpack. Use a loader instead.
 @ ./src/templates.ts 7:37-58
 @ ./src/index.ts 5:20-42

webpack 5.70.0 compiled with 3 warnings in 7474 ms
~~~

Reply:
I get these warnings as well.